psychrochart/chartdata.py

```diff
@@ -304,26 +314,52 @@ def make_constant_enthalpy_lines(
     )
     w_in_sat = _get_humid_ratio_in_saturation(t_sat_points, pressure)
 
-    return PsychroCurves(
-        curves=[
+    curves = [
```

Comment:
@tadashiorigami, I'm not sure about this _hack_ to make enthalpy labels look like this:

![chart_styled_annotations](https://github.com/azogue/psychrochart/assets/9748301/98c21bfd-a8e3-48c8-aa5c-55c0443d7d9a)

🧙‍♂️ It's a pretty obscure logic, IMO, as it requires a lot of trial and error to get the optimal value  for that negative `constant_h_labels_loc` to look good in the plot (**-0.06** in that one), even if you know that the _hack exists_, as the extension of the enthalpy curve in x-axis is calculated as `delta_dry_temp_plot_range * label_loc`.

In addition, it is not filtering out the enthalpy curves without label, so it's printing empty extensions 🐛

I really think that 'feature' of labeling curves like volume, wet-bulb, or enthalpy, with the label located in the saturated zone (with or without dotted-line extension) could be a very nice feature to have, but I'm not sure of how to implement it 😓

Anyway, there is a lot of _black magic_ around the `label_loc` already, so, why not some more 😂

Reply:
> In addition, it is not filtering out the enthalpy curves without label, so it's printing empty extensions 🐛

Fixed it 👍